.github/workflows/pull_request.yml

```diff
@@ -29,13 +29,13 @@ jobs:
         uses: gradle/gradle-build-action@v2.2.0
         if: matrix.os != 'windows-latest'
         with:
-          arguments: --full-stacktrace build
+          arguments: --full-stacktrace assemble
```

Comment:
are we still able to run tests in CI with this configuration?

Reply:
No, the idea is publishing an alpha to try this one, and later when the alpha is available, revert the commit as we talked a few days ago

cc @nomisRev @raulraja

Reply:
Perfect, thanks for clarifying.